src/pseudopeople/interface.py

```diff
@@ -155,10 +155,10 @@ def generate_decennial_census(
     :param seed: An integer seed for randomness.
     :param config: An optional override to the default configuration. Can be a path
         to a configuration YAML file or a dictionary.
-    :param year: The year (format YYYY) to include in the dataset. Must be a decennial
-        year (e.g. 2020, 2030, 2040). Will raise a ValueError if there is no data for
-        this year. If None is provided, data for all years are
-        included in the dataset.
+    :param year: The year in which the census was conducted (format YYYY, e.g., 2030). Must be a decennial
+        year (e.g., 2020, 2030, 2040). Will raise a `ValueError` if there is no data for
+        the specified year. Default is 2020. If `None` is passed instead, data for all avaliable years are
```

Comment:
```suggestion
        the specified year. Default is 2020. If `None` is passed instead, data for all available years are
```

and so on in the other docstrings.

Reply:
Good catch, thanks.